lib/slop/options.rb

```diff
@@ -59,7 +59,9 @@ def on(*flags, **config, &block)
 
     # Add a separator between options. Used when displaying
     # the help text.
-    def separator(string)
+    def separator(string = nil)
+      string ||= ""
```

Comment:
I'm curious about why we're not using `def separator(string = "")` to make the defaulting.

Is there a case for having falsy mean "the empty string"?

Reply:
Oh, I've just found it safer that default values are `nil`.  It allows passing e.g. result of some other method as an argument, which could be `nil` and still get the real default value. Probably not so important here, but I've taught myself to use the patter. =)

Do you want me to change it to `""`?

Reply:
Me, personally, as just another user, would prefer to keep any such defensive code on the consuming side (app, not library).

Reply:
Agreed in this case, I'll fix. Another example where `nil` would be nice, is when passing values from Hashes or other options. Avoids need to check on the client side. But as said, this is simpler and clearer with `""`.